PocketKit/Sources/SharedPocketKit/CDNURLBuilder.swift

```diff
@@ -0,0 +1,37 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+
+/// Builds the url of the image from the pocket CDN
+public struct CDNURLBuilder {
```

Comment:
The name CDN is pretty impenetrable to read.
Would something like `PocketURLCache` or the like still work as a name while providing more readability?

Reply:
I thought it was pretty [common](https://en.wikipedia.org/wiki/Content_delivery_network) and understandable. Cache makes me think about something local, wdyt?